doc_admin/AdminManual.adoc

```diff
@@ -1416,6 +1417,24 @@ By default, ALLOWPRINTOPENEDBILL is set to no.
 NOTE: An application restart is required to apply the modified setting.
 
 
+==== 3.1.36 PATIENTPHOTO
+
+The following table shows the default value and the allowed ones:
+
+[cols=",,",options="header",]
+|===
+|Key |Default Value |Valid Values
+|PATIENTPHOTO |DB |DB, filesystem path
+|===
+
+Open Hospital can save profile picture on DB or on filesystem. 
+When it is not specified DB but a non empty string, Open Hospital will search for profile picture in the path specified in PATIENTPHOTO parameter.
+
+By default, PATIENTPHOTO is set to DB.
```

Comment:
Personally I don't like the idea of over loading the single parameter this way.   I think it would be better to have PATIENTPHOTO with values DB and FILE and a second parameter PATIENTPHOTOPATH if the first is set to FILE.